app/src/main/java/com/anytypeio/anytype/ui/home/HomeScreen.kt

```diff
@@ -98,9 +114,13 @@ fun HomeScreen(
     onSeeAllObjectsClicked: (WidgetView.Gallery) -> Unit,
     onCreateObjectInsideWidget: (Id) -> Unit,
     onCreateDataViewObject: (WidgetId, ViewId?) -> Unit,
-    onBackLongClicked: () -> Unit
+    onBackLongClicked: () -> Unit,
+    showTooltip: StateFlow<Boolean>,
+    onTooltipDismissed: () -> Unit,
 ) {
 
+    val isTooltipVisible by showTooltip.collectAsStateWithLifecycle()
```

Comment:
It is better to move the collectAsStateWithLifecycle function to Fragment

Reply:
Moved the collectAsStateWithLifecycle function to the Fragment